fastlane/Fastfile

```diff
@@ -17,4 +17,21 @@ platform :ios do
       reviewers: [:gersonnoboa, :dukhovnyi, :igorkravchenko, :EgorovEI]
     )
   end
+
+  desc "Creates a PR with the podspec for the Widgets SDK.\n\n"\
+    "Usage:\n"\
+    "fastlane ios upload_widgets_sdk_podspec version:1.0.0\n"
+  lane :upload_widgets_sdk_podspec do |options|
+    version = options[:version]
+    branch_name = "ios-sdk-widgets/#{version}"
+
+    sh "scripts/create_widgets_sdk_pr.sh '#{version}' #{branch_name}"
+    create_pull_request(
+      repo: "salemove/glia-ios-podspecs",
+      title: "Add Widgets SDK podspec for version #{version}",
+      head: branch_name,
+      base: "master",
+      reviewers: [:gersonnoboa, :dukhovnyi, :igorkravchenko, :EgorovEI]
```

Comment:
perhaps we need to add Rasmus to reviewers also: `rasmustautsglia`, if you also think so, please add to `upload_core_sdk_podspec` lane also.

Reply:
Asked about it [here](https://salemove.slack.com/archives/CHN6UB7UH/p1661427477757899) to see if we can use teams instead of individual members to make this more scalable.

Reply:
Added tm-mobile-ios [here](https://github.com/salemove/glia-ios-podspecs/pull/50/files#diff-dff4b99d4e651ab9d7597876ec8dbe92aa934e42c0fb55f4aadd6c106fc96688R34)